akismet-privacy-policies.php

```diff
@@ -38,33 +46,49 @@ class Akismet_Privacy_Policies {
 	 * @return \Akismet_Privacy_Policies
 	 */
 	public function __construct() {
-
-		// The plugin is only helpful on german blogs
-		if ( 'de_DE' !== get_locale() ) {
-			return;
-		}
-
 		register_deactivation_hook( __FILE__, array( &$this, 'unregister_settings' ) );
 		register_uninstall_hook( __FILE__, array( 'Akismet_Privacy_Policies', 'unregister_settings' ) );
 
 		add_filter( 'comment_form_defaults', array( $this, 'add_comment_notice' ), 11, 1 );
 		add_action( 'akismet_privacy_policies', array( $this, 'add_comment_notice' ) );
 
-		$options = get_option( 'akismet_privacy_notice_settings' );
-		if ( empty( $options[ 'checkbox' ] ) ) {
-			$options[ 'checkbox' ] = $this->checkbox;
+		$this->languages = get_available_languages();
+		// default language is en_US but get_available_languages
+		// contains translations in .mo files
+		$this->languages[] = 'en_US';
+
+		if ( isset( $_GET[ 'translation' ])) {
+			$this->translation = $_GET[ 'translation' ];
+		} elseif( isset( $_POST[ 'translation' ])) {
+			$this->translation = $_POST[ 'translation' ];
+		} else {
+			$this->translation = get_user_locale();
+		}
+
+		if ( $this->translation != 'en_US' ) {
+			$this->mo = new Mo;
+			$mofile = dirname( __FILE__) . '/languages/akismet-privacy-policies-' . $this->translation . '.mo';
```

Comment:
Need a space after `__FILE__`.

Reply:
Done. There were a couple of other places where I forgot to put a space between openening/ending brackets and the text within the brackets. I will correct them all.